torch/_refs/special/__init__.py

```diff
@@ -45,6 +49,13 @@ def bessel_j1(a: TensorLikeType) -> TensorLikeType:
     return prims.bessel_j1(a)
 
 
+@_make_elementwise_unary_reference(
```

Comment:
Why are the decorators for `erfcx` and `log_ndtr` below so different? `erfcx` supports `out`, too